2.0.0RC7: NPE in com.owncloud.android.ui.fragment.OCFileListFragment$12.doInBackground

```
java.lang.RuntimeException: 
  at android.os.AsyncTask$3.done (AsyncTask.java:309)
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:354)
  at java.util.concurrent.FutureTask.setException (FutureTask.java:223)
  at java.util.concurrent.FutureTask.run (FutureTask.java:242)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1113)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:588)
  at java.lang.Thread.run (Thread.java:818)
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.ui.fragment.OCFileListFragment$12.doInBackground (OCFileListFragment.java:1541)
  at android.os.AsyncTask$2.call (AsyncTask.java:295)
  at java.util.concurrent.FutureTask.run (FutureTask.java:237)
```
area of code is:
```
final FileDisplayActivity fileDisplayActivity = (FileDisplayActivity) getActivity();
fileDisplayActivity.runOnUiThread(new Runnable() {
    @Override
    public void run() {
        fileDisplayActivity.setIndeterminate(false);
    }
});
```

line 1541 is:
```
fileDisplayActivity.runOnUiThread(new Runnable() {
```

So to fix this we need to do a null check imho.

cc @tobiasKaminsky @mario

--- a/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java
+++ b/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java
@@ -1538,12 +1538,16 @@ public class OCFileListFragment extends ExtendedListFragment implements OCFileLi
                         }
 
                         final FileDisplayActivity fileDisplayActivity = (FileDisplayActivity) getActivity();
-                        fileDisplayActivity.runOnUiThread(new Runnable() {
-                            @Override
-                            public void run() {
-                                fileDisplayActivity.setIndeterminate(false);
-                            }
-                        });
+                        if (fileDisplayActivity != null) {
+                            fileDisplayActivity.runOnUiThread(new Runnable() {
+                                @Override
+                                public void run() {
+                                    if (fileDisplayActivity != null) {
+                                        fileDisplayActivity.setIndeterminate(false);
+                                    }
+                                }
+                            });
+                        }
                     }
 
                     return remoteOperationResult.isSuccess();

